NYTPhotoViewer/NYTPhotoCaptionView.m

```diff
@@ -59,7 +59,18 @@ - (void)layoutSubviews {
     [super layoutSubviews];
 
     void (^updateGradientFrame)(void) = ^{
-        self.gradientLayer.frame = self.layer.bounds;
+        if (@available(iOS 11.0, *)) {
```

Comment:
i vote that we set the project deploy on 11 (IPHONEOS_DEPLOYMENT_TARGET="iOS 11.0")

Reply:
or 12

Reply:
but it doesn't have to be in this PR!

Reply:
I agree! Let's address it on a separate PR. 
I also want to update the Swift examples.
[There is an active effort to modernize this library](https://nytimes.slack.com/archives/CM00QV6H1/p1663252866343959?thread_ts=1663187149.157749&cid=CM00QV6H1)